db/migrate/20210114201454_add_exam_registration_to_exam.rb

```diff
@@ -0,0 +1,5 @@
+class AddExamRegistrationToExam < ActiveRecord::Migration[5.1]
+  def change
+    add_reference :exams, :exam_registration, index: true
```

Comment:
This one should be a many to many relation.
The same exam can be in several registrations

Reply:
Done!